Assets/Scripts/Game Controllers/LevelEndTriggerController.cs

```diff
@@ -0,0 +1,13 @@
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class LevelEndTriggerController : MonoBehaviour
+{
+    private void OnTriggerEnter2D(Collider2D collision)
+    {   
+        if(collision.tag == "Player")
```

Comment:
It's recommended to use the `CompareTag` method instead of using `==` for the tag, but you can leave it as is if you want ([documentation](https://github.com/microsoft/Microsoft.Unity.Analyzers/blob/main/doc/UNT0002.md)).